public/assets/cpu.js

```diff
@@ -82,14 +82,24 @@ function CPU(memamt) {
     this.keyLoad = 'load';
     this.keyMem = 'memoryOrder';
 
-    this.instruction_amounts = {
+    // stores the current num of instructions in one clock cycle
+    this.curr_instructions = {
         'arithmetic' : 0,
         'controlTransfer' : 0,
         'store' : 0,
         'load' : 0,
         'memoryOrder' : 0
     }
 
+    // stores the exponential rolling average of each type of instr
+    this.instruction_amounts = {
+        'arithmetic' : [0, 0],
```

Comment:
sinec instruction_amounts and curr_instructions are just JS objects. could we lose the quotes on the keys, and index into the object like 
```
instruction_amounts.arithmetic++; // for example
```

then i don't think we need the constants `keyLoad`, `keyMem`, etc.

and you should be able to get intellisense in VS Code to auto-complete the key names

Reply:
done